### ISSUE TITLE
Stack Icons clutter the mark up 

### ISSUE THREAD

**Issue by hevayo** [wso2-dev-ux/font-wso2#13]
Hi 

It is nice that the icons can be stack and are useful in certain places. But for icons extensively used it creates a problem of cluttering the markup. For example please see bellow which is used in form field helps.

`<span class="icon fw-stack fw-lg" style="font-size:6px">`
`    <i class="fw fw-circle fw-stack-2x"></i>`
`    <i class="fw fw-question fw-stack-1x fw-inverse" style="font-size:8px"></i>`
`</span>`

this code block will repeat in every form field which will make the form markup long and hard to maintain. 

Possible solutions.
1. Create specific icons without stacking them.
2. Write jquery plugins to insert markup.

Regards

**Comment by dakshika** [comment]
IMO we can't create stacked versions of icons for everything.

**Comment by jeradrutnam** [comment]
Stacking icons is an optional feature. And making all the stacking icon scenarios with single icons will not going to be scalable as well. :)

**Comment by hevayo** [comment]
I was suggesting to create stacked icons for most commonly used icons, for example the question mark with the circle is used in lot of places.  I agree we cannot create stacked versions for all the icons but when you themed a product you tend to see which icons will be mostly stacked. :)

**Comment by jeradrutnam** [comment]
Yes agreed Jo. we were also thinking about it. I'll check if I can write a helper class for this purpose.

**Comment by jeradrutnam** [comment]
Added this feature and will available in the 1.0.3 release. :)

Usage: `<i class="fw fw-view fw-helper fw-helper-circle-outline"></i>`

Currently Available Options:
```
fw-helper-circle-outline
fw-helper-circle
fw-helper-square-outline
fw-helper-square
```

Cheers!
